corehq/apps/importer/templates/importer/partials/import_status.html

```diff
@@ -80,8 +80,12 @@
         </p>
     {% endif %}
     {% if progress.percent %}
-        <div class="progress progress-striped active">
-            <div class="bar" style="width: {{ progress.percent }}%;"></div>
+        <div class="progress">
+            <div class="progress-bar progress-bar-info"
+                 role="progressbar" aria-valuenow="60" aria-valuemin="0" aria-valuemax="100"
```

Comment:
it also doesn't hurt to have them in, and we have those present in a number of places.